docs/merged_objects.md

```diff
@@ -224,8 +224,106 @@ Some merged objects may have some additional sample metadata columns specific to
+
+
+In addition, the following QC statistics from [`DropletUtils::cleanTagCounts()`](https://rdrr.io/github/MarioniLab/DropletUtils/man/cleanTagCounts.html), which were calculated on individual libraries before objects were merged, can be found in the `colData` of the `"adt"` alternative experiment, accessed with `colData(altExp(merged_sce, "adt"))`.
+
+| Column name                | Contents                                          |
+| -------------------------- | ------------------------------------------------- |
+| `zero.ambient`   | Indicates whether the cell has zero ambient contamination   |
+| `sum.controls` |  The sum of counts for all control features. Only present if negative/isotype control ADTs were used |
+| `high.controls`  | Indicates whether the cell has unusually high total control counts. Only present if negative/isotype control ADTs were used |
+| `ambient.scale` |  The relative amount of ambient contamination. Only present if negative/isotype control ADTs were not used |
+| `high.ambient`  | Indicates whether the cell has unusually high contamination. Only present if negative/isotype control ADTs were not used |
+| `discard`  | Indicates whether the cell should be discarded based on ADT QC statistics |
+
+
+Metrics for each of the ADTs assayed can be found as a `DataFrame` stored as `rowData` within the alternative experiment:
+
+```r
+rowData(altExp(merged_sce, "adt")) # adt metrics
+```
+
+This data frame contains the following columns with statistics for each ADT:
+
+| Column name | Contents                                                       |
+| ----------- | -------------------------------------------------------------- |
+| `mean-SCPCL000000`      | Mean ADT count across all cells/droplets                       |
+| `detected-SCPCL000000`  | Percent of cells in which the ADT was detected (ADT count > 0 ) |
+| `target_type` | Whether each ADT is a target (`target`), negative/isotype control (`neg_control`), or positive control (`pos_control`). If this information was not provided, all ADTs will have been considered targets and will be labeled as `target` |
+
+Finally, additional metadata for ADT processing can be found in the metadata slot of the alternative experiment
+
+Metadata associated with {ref}`data processing <processing_information:Processing information>` is included in the `metadata` slot as a list.
+Similar to the [parent experiment metadata](#singlecellexperiment-experiment-metadata), the `metadata` will contain three items: `library_id`, `sample_id`, and `library_metadata`.
+The `library_metadata` contains the same list of metadata as its corresponding parent experiment library's metadata and includes one additional field `ambient_profile`, which holds a list of the ambient concentrations of each ADT for the given library.
+
+```r
+metadata(altExp(merged_sce, "adt")) # adt metadata
+```
+
+### Additional SingleCellExperiment components for multiplexed libraries
+
```

Comment:
I think if we are trying to mirror the contents of the [SCE file contents](https://github.com/AlexsLemonade/scpca-docs/blob/sjspielman/240-remaining-sce-sections-merged-object/docs/sce_file_contents.md#additional-singlecellexperiment-components-for-multiplexed-libraries), then we should also mention accessing the cellhash ADT.

Reply:
> then we should also mention accessing the cellhash ADT.

- cellhash altexp?
- I already include a sentence that there is no cellhash altexp in the merged object.

Reply:
yes no cell hash ADT, sorry about that confusion!